plugins/devtools-backend/package.json

```diff
@@ -41,7 +41,7 @@
     "@manypkg/get-packages": "^1.1.3",
     "@types/express": "*",
     "@yarnpkg/lockfile": "^1.1.0",
-    "@yarnpkg/parsers": "^3.0.0-rc.4",
```

Comment:
The DevTools plugin in this repo has been deprecated in favor of the version in the `community-plugins` repo. Given that we should probably not make this change.

Reply:
@awanlin the devtools plugin hasn't been moved :)